suite-native/module-onboarding/src/screens/OnboardingAssets.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { View } from 'react-native';
+import { useDispatch } from 'react-redux';
+
+import { CryptoIcon, Icon } from '@trezor/icons';
+import TrezorConnect, { AccountInfo } from '@trezor/connect';
+import { Screen, StackProps } from '@suite-native/navigation';
+import {
+    Box,
+    Button,
+    Text,
+    Card,
+    InputWrapper,
+    Input,
+    SelectableListItem,
+    VStack,
+} from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { OnboardingStackParamList, OnboardingStackRoutes } from '../navigation/routes';
+import { AssetsLoader } from '../components/AssetsLoader';
+import { setOnboardingFinished } from '../slice';
+
+const assetsStyle = prepareNativeStyle(utils => ({
+    flex: 1,
+    padding: utils.spacings.medium,
+    justifyContent: 'space-between',
+}));
+
+export const OnboardingAssets = ({
+    route,
+}: StackProps<OnboardingStackParamList, OnboardingStackRoutes.OnboardingAssets>) => {
+    const dispatch = useDispatch();
+    const [radioChecked, setRadioChecked] = useState<string>('firstSelectable');
+    const [inputText, setInputText] = useState<string>('');
+    const [accountInfo, setAccountInfo] = useState<AccountInfo | null>(null);
+    const { applyStyle } = useNativeStyles();
+
+    const { xpubAddress, currencySymbol } = route.params;
+
+    const getAccountInfo = useCallback(() => {
+        TrezorConnect.getAccountInfo({
+            coin: currencySymbol,
+            descriptor: xpubAddress,
+        })
+            .then(accountInfo => {
+                if (accountInfo?.success) {
+                    setAccountInfo(accountInfo.payload);
+                }
+                // TODO error?
+            })
+            .catch(error => {
+                // eslint-disable-next-line no-console
+                console.log('getAccountInfo failed: ', JSON.stringify(error));
+            });
+    }, [xpubAddress, currencySymbol]);
+
+    useEffect(() => {
+        const fetchingTimerId = setTimeout(() => {
+            getAccountInfo();
+        }, 800);
```

Comment:
Why use setTimeout? Is it related to the splash screen?

Reply:
I was trying to simulate loading according to Figma.

![Snímek obrazovky 2022-08-04 v 10 08 03](https://user-images.githubusercontent.com/2227592/182797091-0eacb282-eb5e-46fb-a814-eb8b8657cba8.png)

Reply:
So if I understand correctly, you are using the splash screen (where same 800 timeout is set) and here you are waiting to fetch until this screen disappears?

Reply:
Note: After call with @bouchja1, setTimeout is present only to allow for loading screen to be visible for 800ms. As discussed, it will be resolved with just comment as this will probably be handled by extra builders from redux toolkit when it's implemented in thunks.

Reply:
added as we discussed https://github.com/trezor/trezor-suite/pull/5869/commits/b12e5e60dc7812a950166eda5cbf4e9a4cfd34ca